fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -142,22 +142,27 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	//   is possible within but not between priority groups; fewer priority groups
 	//   mean better worst case time bounds
 	// Maximum allowable priority is 999.
-	int PRIORITY_RECOVER_MOVE;
-	int PRIORITY_REBALANCE_UNDERUTILIZED_TEAM;
-	int PRIORITY_REBALANCE_OVERUTILIZED_TEAM;
-	int PRIORITY_REBALANCE_READ_OVERUTIL_TEAM;
-	int PRIORITY_REBALANCE_READ_UNDERUTIL_TEAM;
-	int PRIORITY_PERPETUAL_STORAGE_WIGGLE;
-	int PRIORITY_TEAM_HEALTHY;
-	int PRIORITY_TEAM_CONTAINS_UNDESIRED_SERVER;
-	int PRIORITY_TEAM_REDUNDANT;
+	// Update the status json .data.team_tracker.state field when necessary
+	int PRIORITY_RECOVER_MOVE; // Priority for movement resume from previous unfinished in-flight movement when a new DD
```

Comment:
Could we put the comments above the priorities? The formatter does unpleasant things with comments that are placed at the end of lines

Reply:
I guess it didn't actually do anything bad in this file, probably because the source part is all reasonably short. I'll leave it up to you, though generally I prefer to avoid this style altogether because of the potential for it to split up the source part of the line across multiple lines when there is a comment.